assets/src/stories-editor/constants.js

```diff
@@ -5,6 +5,7 @@ import { __ } from '@wordpress/i18n';
 
 export const STORY_PAGE_INNER_WIDTH = 328;
 export const STORY_PAGE_INNER_HEIGHT = 553;
+export const STORY_PAGE_MARGIN = 50;
```

Comment:
Just realized there is `TOTAL_PAGE_MARGIN` in the `EditorCarousel` component:

https://github.com/ampproject/amp-wp/blob/86370162531938d30d518ae9d552d6ba48a3b8ff/assets/src/stories-editor/components/editor-carousel/index.js#L23-L24

Let's use `STORY_PAGE_MARGIN` there too 🎉

Reply:
Fixed in 64f341d.